### PR TITLE
feat(#1658): Add flag to skip habitat installation

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

We need to have a flag to allow skipping habitat installation for AWS Integration setup

## Objective

This PR adds a flag to skip habitat processing in the launcher and update `launch` command for AWS integration compatibility

## References

https://github.com/screwdriver-cd/screwdriver/issues/1658

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: Docker/launcher_entrypoint.sh [hunk 1]
```diff
@@ -52,26 +52,29 @@ smart_run whoami
...
-
-# Create directory for hab pkg binlink destination
-smart_run mkdir -p /usr/sd/bin
-
-# Binlinking bash from core/bash into /bin
-if ! binary_exists bash; then
-    smart_run /opt/sd/bin/hab pkg binlink core/bash bash || echo 'Failed to symlink bash'
-fi
+
+if ! $SD_NO_HAB; then
```
Comment (jithine) [inline Docker/launcher_entrypoint.sh:56]: can we call it `SD_HAB_ENABLED` and this check shouild change to `if [ "$SD_HAB_ENABLED" = "yes" ];`

Comment (pritamstyz4ever) [reply Docker/launcher_entrypoint.sh:56]: ok, doing the inverse means, we need to change code in all executors again to make it work

Comment (jithine) [reply Docker/launcher_entrypoint.sh:56]: I feel it's more readable with `SD_HAB_ENABLED`, it took me a while to write the if logic . with inverse. let's update executors.

Comment (pritamstyz4ever) [reply Docker/launcher_entrypoint.sh:56]: Updated the flag name!

Comment (tkyi) [reply Docker/launcher_entrypoint.sh]: can we set to `true` vs `yes` just to be consistent?

**Review by jithine — APPROVED** [review]

**Pull request merged**